ecs-cluster.tf

```diff
@@ -1,3 +1,9 @@
+resource "aws_ecs_cluster_capacity_providers" "atlantis" {
+  cluster_name = var.create_ecs_cluster ? aws_ecs_cluster.atlantis[0].name : var.ecs_cluster_name
+
+  capacity_providers = ["FARGATE"]
```

Comment:
Any reason spot capacity providers aren't supported?

Reply:
Yeah, I will implement this at a later stage. I first have to implement persistent storage.

Reply:
Ok 👍🏼